internal/graph/graph_test.go

```diff
@@ -21,16 +21,16 @@ var (
 
 		// Sort by Type and then by edge and then by tupleset relation
 		sort.SliceStable(out, func(i, j int) bool {
-			if out[i].Type > out[j].Type {
-				return false
+			if out[i].Type != out[j].Type {
+				return out[i].Type < out[j].Type
 			}
```

Comment:
this problem was identified 3 weeks ago 😅 https://github.com/openfga/openfga/pull/1299#discussion_r1446703961

Reply:
🙈

Reply:
Ideally we'd delegate to a shared test utility here that shares the logic. Otherwise this may bite us in the ass again.